org.eclipse.wb.core/src/org/eclipse/wb/core/controls/palette/PaletteComposite.java

```diff
@@ -63,21 +60,16 @@ public final class PaletteComposite extends Composite {
   private static final Color COLOR_PALETTE_BACKGROUND = IColorConstants.button;
   private static final Color COLOR_TEXT_ENABLED = IColorConstants.listForeground;
   private static final Color COLOR_TEXT_DISABLED = IColorConstants.gray;
-  private static final Color COLOR_ENTRY_SELECTED = DrawUtils.getShiftedColor(
-      COLOR_PALETTE_BACKGROUND,
-      24);
-  private static final Color COLOR_CATEGORY_GRAD_BEGIN = DrawUtils.getShiftedColor(
-      COLOR_PALETTE_BACKGROUND,
-      -8);
-  private static final Color COLOR_CATEGORY_GRAD_END = DrawUtils.getShiftedColor(
-      COLOR_PALETTE_BACKGROUND,
-      16);
-  private static final Color COLOR_CATEGORY_SEL_GRAD_BEGIN = DrawUtils.getShiftedColor(
-      COLOR_PALETTE_BACKGROUND,
-      16);
-  private static final Color COLOR_CATEGORY_SEL_GRAD_END = DrawUtils.getShiftedColor(
-      COLOR_CATEGORY_GRAD_BEGIN,
-      -8);
+  private static final Color COLOR_ENTRY_SELECTED =
+      DrawUtils.getShiftedColor(COLOR_PALETTE_BACKGROUND, 24);

```

Comment:
Undesired, looks like the code formatter uses short line breaks. We should change this via another PR.